modules/log6x-rn-6.0.6.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-6_{context}"]
+= Logging 6.0.6
+This release includes link:https://access.redhat.com/errata/RHSA-2025:3132[RHSA-2025:3132].
+
+[id="logging-release-notes-6-0-6-bug-fixes_{context}"]
+== Bug Fixes
+* Before this update, the Operator deployed the collector with output configurations that are not referenced by any inputs. With this update, the Operator adds validation to fail the `ClusterLogForwarder` if an output configuration is not referenced by any inputs, preventing the deployment of the collector.
```

Comment:
```suggestion
* Before this update, the Logging Operator deployed the collector config map with output configurations that were not referenced by any inputs. With this update, the Operator adds validation to fail the `ClusterLogForwarder` custom resource if an output configuration is not referenced by any inputs, preventing the deployment of the collector.
```
I based this revision on information I saw in the related bug. Feel free to revise or ignore my suggestions here if they are inaccurate.